lib/aerosol/instance.rb

```diff
@@ -25,7 +25,7 @@ def private_ip_address
   end
 
   def address
-    if public_hostname.blank?
+    if launch_configuration.try(:always_use_private_address) || public_hostname.blank?
```

Comment:
I assumed I was breaking some invariant, but maybe it was just protecting bad stubs. Likely that.